package.json

```diff
@@ -43,7 +43,7 @@
     "mocha": "^3.2.0",
     "temp": "^0.8.1",
     "tslint": "^3.15.1",
-    "typescript": "2.2.1"
+    "typescript": "2.3.0"
```

Comment:
Done. I removed the leading `^`, as it didn't make sense to make it different from the other version specifier.
Ran yarn, and nothing was modified.